src/qibocal/protocols/characterization/allxy/drag_pulse_tuning.py

```diff
@@ -28,6 +28,10 @@ class DragPulseTuningParameters(allxy_drag_pulse_tuning.AllXYDragParameters):
     """DRAG pulse beta end sweep parameter."""
     beta_step: float
     """DRAG pulse beta sweep step parameter."""
+    qubits: list = field(default_factory=list)
+    """Local qubits (optional)."""
+    update: Optional[bool] = None
```

Comment:
Are you sure that it shouldn't be:
```suggestion
    update: bool = False
```
as for the others?